frontends/web/src/routes/new-settings/components/appearance/activeCurrenciesDropdownSetting.tsx

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
+import { useEffect, useState } from 'react';
 import Select, { ActionMeta, MultiValue, MultiValueRemoveProps, components } from 'react-select';
 import { currencies, store, selectFiat, unselectFiat, SharedProps } from '../../../../components/rates/rates';
 import { SettingsItem } from '../settingsItem/settingsItem';
 import { Fiat } from '../../../../api/account';
-import styles from './defaultCurrencySetting.module.css';
```

Comment:
delete file?

Reply:
Haha, nice. Thanks. Funny I didn't delete the file at all while mentioning that I did so in the commit msg 😄